src/node.cc

```diff
@@ -3334,6 +3338,11 @@ void SetupProcessObject(Environment* env,
     READONLY_PROPERTY(process, "_debugWaitConnect", True(env->isolate()));
   }
 
+  // --expose_internals,--expose-internals
+  if (expose_internals) {
+    READONLY_PROPERTY(process, "_exposeInternals", True(env->isolate()));
+  }
```

Comment:
Would prefer to use `process.binding('config')` for this rather than adding a new `_`-prefixed property off `process`